src/math/euler_totient.rs

```diff
@@ -0,0 +1,45 @@
+/// ## Euler's Totient (`φ(n)`)
+///
+/// Calculate the **Euler's Totient** function of a given number `n`.
+///
+/// Wikipedia: https://en.wikipedia.org/wiki/Euler%27s_totient_function
+pub fn euler_totient(mut n: i64) -> i64 {
```

Comment:
What do you think about having the following signature:
```suggestion
pub fn euler_totient(n: u64) -> Result<u64, &'static str> {
```